com/stencyl/models/IdType.hx

```diff
@@ -0,0 +1,11 @@
+package com.stencyl.models;
+
+typedef IdType = Int
+
+class IdUtils {
+	public static var INVALID_ID = -1;
+
+	public static function parseXmlId(id: String): IdType {
+		return Std.parseInt(id);
+	}
```

Comment:
`parseXmlId` is a little weird to me because it's not parsing from `Fast` or `Xml`, but `String`. Simple `parseId` might be better.